src/card.css

```diff
@@ -0,0 +1,77 @@
+.card {
+  padding: 16px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  display: flex;
+  height: 134px;
+  width: 160px;
+  padding: 16px 3px 40px;
+}
+
+.card__description {
+  font-size: 12px;
+  font-weight: 500;
+  margin-bottom: 4px;
+  line-height: 18px;
+}
+
+.card__code {
+  font-size: 10px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.reviews {
+  font-size: 10px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  margin-bottom: 16px;
+}
+
+.price-text {
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  color: #616070;
+}
+
+.price-tag {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card__buy-button {
+  font-size: 14px;
+  font-weight: 700;
+  background-color: #00acdc;
+  line-height: 16px;
```

Comment:
Organize your styles
I recommend you take a look at these styles code and use it in the feature 
[9elements](https://9elements.com/css-rule-order) or  [htmlAcademy](https://codeguide.academy/html-css.html#css-order)